packages/common-scripts/examples/pw_lock/config.json

```diff
@@ -0,0 +1,49 @@
+{
```

Comment:
You can add a `__COMMENT__` field, describing how this file is configured, such as:

* Is it a testnet or a devnet config?
* How can I use it?

Reply:
Updated.